### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by kasionio**
### html-css-popup

[Demo](https://kasionio.github.io/html-css-popup)
[Code base](https://github.com/kasionio/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/kasionio/html-css-popup/index.css [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+.nav-bar {
+	width: 250px;
+	display: inline-flex;
+	justify-content: space-around;
+	align-items: center;
+	float: right;
+	padding: 15px;
+}
+
+#nav-link__images {
```
Comment (A-Ostrovnyy) [inline submissions/kasionio/html-css-popup/index.css:21]: Good Work!
Avoid styling by #id or HTML tag. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, separate responsibilities:

- HTML tag for markup;
- CSS class for styling;
- Id for JS logic

File: submissions/kasionio/html-css-popup/index.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+	object-fit: contain;
+}
+
+#popup-menu__button {
+	position: fixed;
+	right: 105px;
+	appearance: none;
+	width: 25px;
+    height: 25px;
+    background: url(popup-icons/popup-button.png) no-repeat;
```
Comment (A-Ostrovnyy) [inline submissions/kasionio/html-css-popup/index.css:37]: Use [Prettier](https://prettier.io/) for auto-formatting your code

File: submissions/kasionio/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="index.css">
+      <link rel="shortcut icon" type="image/x-icon" href="./popup-icons/favicon.png" />
+      <title>Hooli-style Popup</title>
+   </head>
+   <body>
+      <nav>
+         <div class="nav-bar">
+            <a href="#mail">Mail</a>
+            <a href ="#images" id='nav-link__images'>Images</a>
```
Comment (A-Ostrovnyy) [inline submissions/kasionio/html-css-popup/index.html:14]: For menu component  preferable markup is:
```HTML
<nav>
    <ul>
        <li></li>
        <li></li>
        <li></li>
    </ul>
</nav>
```

File: submissions/kasionio/html-css-popup/index.css [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+	justify-content: space-around;
+	flex-wrap: wrap;
+}
+
+#popup-menu-more__button:checked,
+#popup-menu-more__button:checked~.popup-menu-more__text {
+	display: none;
+}
+
+#divide-line {
```
Comment (A-Ostrovnyy) [inline submissions/kasionio/html-css-popup/index.css:150]: Avoid adding visual elements to the markup, use CSS(border-top, border-bottom, or pseudoelements) for it

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good job!

File: submissions/kasionio/html-css-popup/index.css (hunk 1 above)
Comment (kasionio) [reply submissions/kasionio/html-css-popup/index.css]: Thanks a lot for the review! Changes are made

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**